packages/steveo/src/consumers/sqs.ts

```diff
@@ -71,14 +71,17 @@ class SqsRunner extends BaseRunner implements IRunner {
               return;
             }
 
+            // const { _meta, context, ...data } = c.payload;
+            const data = { ...c.payload };
+            delete data.context;
```

Comment:
Why are we doing the clone and delete instead of
```
const { _meta, context, ...data } = c.payload;
```
?

Reply:
Thinking about this.. is there a risk that we drop a field called `context` from a message?

Reply:
the reason i originally used _meta was to avoid conflict with a parameter that would be used in an actual payload. 

you see usage of `$` to prefix internal parameters also seems like a reasonable decision.

regardless 

should the change here be just 

const data = c.payload;

we don't need to spread to clone the internal payload here do we?